queries.sql

```diff
@@ -231,4 +231,114 @@ FROM
 GROUP BY
     owners.full_name
 ORDER BY
-    total_animals DESC;
\ No newline at end of file
+    total_animals DESC;
+
+/* JOIN tables queries */
+SELECT
+    animals.name AS animals_name,
+    visits.visit_date
+FROM
+    animals
+    JOIN visits ON animals.id = visits.animals_id
+    JOIN vets ON vets.id = visits.vets_id
+WHERE
+    vets.name = 'William Tatcher'
+ORDER BY
+    visits.visit_date DESC
+limit
+    1;
+
+SELECT
+    COUNT(animals.name) AS Total_no_of_animals
+FROM
+    animals
+    JOIN visits ON animals.id = visits.animals_id
+    JOIN vets ON vets.id = visits.vets_id
+WHERE
+    vets.name = 'Stephanie Mendez';
+
+SELECT
+    vets.name,
+    species.name AS specialization
+FROM
+    vets
+    LEFT JOIN specializations ON specializations.vets_id = vets.id
+    LEFT JOIN species ON species.id = specializations.species_id;
+
+SELECT
+    animals.name AS animals_name
+FROM
+    animals
+    JOIN visits ON animals.id = visits.animals_id
+    JOIN vets ON vets.id = visits.vets_id
+WHERE
+    vets.name = 'Stephanie Mendez'
+    AND visits.visit_date BETWEEN '2020-04-01'
+    AND '2020-08-30';
+
+SELECT
+    animals.name AS animals_name,
+    COUNT(visits.visit_date) AS total_visits
+FROM
+    animals
+    JOIN visits ON animals.id = visits.animals_id
+    JOIN vets ON vets.id = visits.vets_id
+GROUP BY
+    animals.id
+ORDER BY
+    total_visits DESC
+limit
+    1;
```

Comment:
- [ ] Here we have to find, What animal has the most visits to vets?
```
  SELECT animals.name, count(animals.name)
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  ORDER BY count(animals.name) DESC
```

Reply:
I will update this too😉